bin/cdnjs-index.js

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env node
+global._ = require('lodash'); // ¯\_(ツ)_/¯
+
+const path = require('path');
+const tar = require('tar-stream');
+const config = require('config');
+const zlib = require('zlib');
+const got = require('got');
+const micromatch = require('micromatch');
+const Bluebird = require('bluebird');
+
+const pipeline = require('util').promisify(require('stream').pipeline);
+
+const db = require('../src/lib/db/index.js');
+const CndJsPackage = require('../src/models/CdnJsPackage');
+
+const httpClient = got.extend({ headers: { 'user-agent': config.get('server.userAgent') }, json: true });
```

Comment:
The global `json: true` seems confusing, especially as you later request binary data with this. It's ignored for streaming mode but still, let's put json only when it's needed.

Reply:
It doesn't seems confusing for me. Every remote source requested by this client is expected to return a json. So it makes sense to set it once and to disable it when needed. But again in context of this script no other types are expected. Except stream of course, but as you said this is ignored for streams.